item.rb

```diff
@@ -0,0 +1,35 @@
+class Item
+  attr_accessor :genre, :author, :label, :publish_date
+
+  def initialize(publish_date, id = nil, archived: false)
+    @id = id || rand(1_000_000)
+    @publish_date = publish_date
+    @archived = archived
+  end
+
+  def add_genre(genre)
+    @genre = genre
+  end
+
+  def add_author(author)
+    @author = author
+  end
+
+  def add_source(source)
+    @source = source
+  end
+
+  def add_label(label)
+    @label = label
+  end
+
```

Comment:
- [ ] @Team great work on your item class, however, you forgot to add the 1-to-many relationship between the genre author and label classes to item classes, below is an example of how to accomplish it eg
 ```
 def genre=(genre)
      @genre = genre
      genre.items.push(self) unless genre.items.include?(self)
    end
```